qiskit_nature/operators/second_quantization/particle_op.py

```diff
@@ -12,44 +12,48 @@
 
 """The Sum Operator base interface."""
 
-import copy
 from abc import ABC, abstractmethod
-from typing import Any, List
 
+from qiskit.opflow import PauliSumOp
 
-class SumOp(ABC):
-    """The Sum Operator base interface.
+from .star_algebra import StarAlgebraMixin
+from .tolerances import TolerancesMixin
+
+
+class ParticleOp(StarAlgebraMixin, TolerancesMixin, ABC):
+    """The Second Quantized Operator base interface.
 
     This interface should be implemented by all creation- and annihilation-type particle operators
     in the second-quantized formulation.
     """
 
-    def __init__(self, particle_type):
-        self._particle_type = particle_type
-
-    @property
-    def particle_type(self):
-        """Return the particle type"""
-        return copy.deepcopy(self._particle_type)
-
     @property
     @abstractmethod
     def register_length(self) -> int:
```

Comment:
~~I guess we can only remove this once we have also removed it from `BosonicOp` and `SpinOp`, right?~~

EDIT: sorry, I didn't realize we only removed it from `__init__`. Do we actually still need it though? The length of the label should be sufficient, right?

Reply:
Basically, if you take the length of the label, you can get it as information. However, the shape (#terms, lem (label)) of the operator is often used, so I think it is better to have the method or property. I think there is room for renaming, but would be better to have such a function.